client/blocks/comment-likes/index.jsx

```diff
@@ -15,19 +15,20 @@ import { likeComment, unlikeComment } from 'state/comments/actions';
 import { getCommentLike } from 'state/comments/selectors';
 
 const getLabel = ( i_like, like_count, translate ) => {
+	if ( i_like && like_count === 1 ) {
+		return translate( 'Liked', { comment: 'Displayed when a person "likes" a post.' } );
```

Comment:
I know that the comments are generally safe here in the translations and I wonder if we need it.

On the other hand, this would definitely make it easier for people to translate gender, since they will presumably know what gender a post is.

@yoavf this might be a good example of a situation like we spoke about. do you have any insightful comment here?